test/basics/demo-complex.html

```diff
@@ -0,0 +1,18 @@
+<script type='text/stache' can-autorender id='demo-html'>
+	{{#each items}}
+		<li>{{name}}</li>
+	{{/each}}
+</script>
+</script>
+
+<script src="../../node_modules/steal/steal.js" main="@empty" id='demo-source'>
+import "can-view-autorender";
+import "can-stache-bindings";
```

Comment:
There was a `(click)` handler that was causing errors under zombie that I removed, so that explains where the `can-stache-bindings` came from. The error had to do with `cloneNode`, but don't think that's something we would want to resolve just to support zombie. I took this from the `canjs/canjs` repo, so it's a real example, which I thought would be important to include next to the overly simple examples. The deps are that big of a burden since they're small and quick to install, and are dev only, but I can certainly simplify it if we're not losing anything beneficial by doing so.